setup.cfg

```diff
@@ -29,7 +29,7 @@ install_requires =
     xarray
     pyqtgraph>=0.12.1
     matplotlib>=3.0.0
-    numpy>=1.12.0
```

Comment:
I don't think its a good idea to limit the min version to 2.0.0 according to the docs's https://numpy.org/doc/1.13/reference/generated/numpy.issubdtype.html issubdtype has been available since at lease numpy 1.13.0 so I think it would be safe to limit this to something like that

Reply:
issubdtype also exists in numpy 1.12: https://docs.scipy.org/doc/numpy-1.12.0/reference/generated/numpy.issubdtype.html

Numpy 1.12.0 should still work so I will discard these changes